app/views/shared/_brand.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class='logo'>
```

Comment:
Did you merge this with the redesign branch? I refactored this partial the other day